src/libs/actions/IOU.js

```diff
@@ -720,6 +720,93 @@ function payIOUReport({
     return promiseWithHandlers;
 }
 
+/**
+ * @param {Object} report
+ * @param {Number} amount
+ * @param {String} currency
+ * @param {String} comment
+ * @param {String} ownerEmail - Email of the person generating the IOU.
+ * @param {String} userEmail - Email of the other person participating in the IOU.
+ */
+function sendMoneyElsewhere(report, amount, currency, comment, ownerEmail, userEmail) {
+    const participants = [
+        {login: ownerEmail},
+        {login: userEmail},
+    ];
+    const chatReport = lodashGet(report, 'reportID', null)
+        ? report
+        : ReportUtils.buildOptimisticChatReport(participants);
+
+    const chatReportID = chatReport.reportID;
```

Comment:
Here we also need ot use the same logic as here https://github.com/autosave-app/App/blob/2356010ff7817ae27e603558b6905474929465d6/src/libs/actions/IOU.js#L59-L65

to see if we need to create a new optimistic chat.